README.md

```diff
@@ -1,24 +1,143 @@
-# README
+<a name="readme-top"></a>
+<!-- TABLE OF CONTENTS -->
+<div align="center">
+  
+</div>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
 
-Things you may want to cover:
+# 📗 Table of Contents
 
-* Ruby version
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
 
-* System dependencies
+- [📝 License](#license)
 
-* Configuration
 
-* Database creation
+# 📖 Budget App <a name="about-project"></a>
 
-* Database initialization
+**Budget App** is a Ruby on Rails capstone project which you can create new categories, you can write new transactions and deploy it the categories and finally you can control how much money you spend each category.
 
-* How to run the test suite
+## Video Link 🚀
 
```

Comment:
- [ ] Thank you for communicating your audio issues in the PR summary. However, I'm afraid you'll have to work things around a little bit in order to meet the project requirements. Anyways, here are some steps to check in order to fix your audio problems in Ubuntu OS:

1. Install and Use pavucontrol
pavucontrol is a PulseAudio Volume Control application that provides more advanced control over audio settings. Install it and check the microphone settings.

2. Check PulseAudio Configuration
`sudo nano /etc/pulse/default.pa`

3. Restart PulseAudio
```
pulseaudio -k
pulseaudio --start
```

4. Check for System Updates
```
sudo apt-get update
sudo apt-get upgrade
```

5. Reinstall Audio Drivers
If none of the above steps work, you may want to consider reinstalling audio drivers.

Reply:
Needs attention